### PR TITLE
Page attachment call to action animation and design review.

### PR DESCRIPTION
**Pull request by gmajoulet**
Page attachment call to action animation and design review.

- New animations: the arrow starts as a flat line and turns into an arrow (super subtle tho), and then "explodes" into a button. The text is no longer a button.
- Refining the text shadows, the background gradient, for better accessibility. It reads OK even on a full white background
- The arrow is now clickable/tappable

Demo on page two available until this PR is merged: [here](https://stamp-gmajoulet.firebaseapp.com/examples/s20/body-painting/index.html)

### CODE REVIEW

**Comment by Enriqe** [comment]
Looks pretty cool! 

Do we want to restart the animation every time we visit the page containing the attachment? Playing with the demo I skipped the page, then came back and the animation was already over. Just thinking out loud, but otherwise it lgtm!

File: extensions/amp-story/1.0/amp-story.css [hunk 1]
```diff
@@ -816,64 +816,105 @@ amp-story .amp-video-eq,
...
+amp-story-page .i-amphtml-story-page-open-attachment-icon::after {
+  content: "" !important;
   position: absolute !important;
   top: 0 !important;
   left: 0 !important;
   height: 100% !important;
   width: 100% !important;
-  background: #fff !important;
-  border-radius: 32px !important;
+  background: #FFF;
```
Comment (newmuis) [inline extensions/amp-story/1.0/amp-story.css]: `!important`?

Comment (gmajoulet) [reply extensions/amp-story/1.0/amp-story.css]: Yep :))

**Comment by gmajoulet** [comment]
Good point, thanks! Hong said we should play the animation every time the page becomes active, I just made the change. PTAL

**Pull request merged**